### PR TITLE
Mp 1621 liquidation filter deploy scripts

### PR DESCRIPTION
**Pull request by brimigs**
This PR includes: 
- deploy scripts for liquidation filter contract 
- allows for the owner to be set as either the deployer address or the multisig address 
- updated README with deployment steps 
- adds in ts codegen 
- add in prettierrc 
- adds in linter

### CODE REVIEW

File: scripts/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "singleQuote": true,
+    "jsxSingleQuote": true,
+    "semi": false,
+    "printWidth": 100,
+    "trailingComma": "all"
+  }
+
```
Comment (grod220) [inline scripts/.prettierrc:8]: A couple of files are missing the new lines at the end of them. On the PR you can see by the 🚫 icon.

File: scripts/deploy/base/index.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    await deployer.assertDeployerBalance()
+
+    // Upload contracts
+    await deployer.upload('liquidationFilterer', 'mars_liquidation_filterer.wasm')
+
+    // Instantiate contracts
+    await deployer.instantiateLiquidationFilterer()
+    await deployer.saveDeploymentAddrsToFile()
+
+    //update owner to multisig address
```
Comment (piobab) [inline scripts/deploy/base/index.ts:19]:
```suggestion
    // update owner to multisig address
```

**Review by piobab — APPROVED** [review]
LGTM

**Review by grod220 — APPROVED** [review]
LGTM, just add those new lines plz

File: scripts/.prettierrc (hunk 1 above)
Comment (brimigs) [reply scripts/.prettierrc]: Oops I keep forgetting to run format before my PR.. just updated :)

**Pull request merged**